src/leiningen/v/git.clj

```diff
@@ -7,32 +7,52 @@
 (def ^:dynamic *min-sha-length* 4)
 (def ^:dynamic *dirty-mark* "DIRTY")
 
-(let [shell "/bin/bash"
-      cmd [shell "-c"]]
-  (defn- git-command
-    [command]
-    (let [cmd (conj cmd (str "git " command))
-          {:keys [exit out err]} (apply shell/sh cmd)]
-      (if (zero? exit)
-        (string/split-lines out)
-        (do (lein/warn err) nil)))))
+(def unix-git-command "git")
+
+(def ^:private windows?
```

Comment:
This fn simply reads the env -in such cases I prefer that the var reference data read at startup instead of a runtime invoked fn.

Reply:
Thanks for looking at this. Latest commit updates this to run at load time, and I'm using @empperi's more general Windows regex.